File: aspectj(java)/src/main/java/com/tend/model/Account.java
package com.tend.model;

import com.tend.Main;

public class Account {
    public int balance = 20;

    public boolean withdraw(int amount) {
        if (balance < amount) {
            System.out.println(Main.ANSI_YELLOW + "[ACTUAL FUNCTION] Not enough balance" + Main.ANSI_RESET);
            return false;
        }
        balance -= amount;
        System.out.println(Main.ANSI_YELLOW +  String.format("[ACTUAL FUNCTION] Widraw %s", amount + Main.ANSI_RESET));
        return true;
    }

    public boolean deposit(int amount) {
        balance += amount;
        return true;
    }
}

File: aspectlib(python)/Account.py
class Account():
    ANSI_YELLOW = "\033[33m"
    ANSI_RESET = '\033[0m'

    def __init__(self):
        self.balance = 20

    def withdraw(self, amount: int) -> bool:
        if (self.balance < amount):
            print(f"{self.ANSI_YELLOW}[ACTUAL FUNCTION] Not enough balance |  Balance left: {self.balance}{self.ANSI_RESET}")
            return False

        self.balance -= amount
        print(f"{self.ANSI_YELLOW}[ACTUAL FUNCTION] Widraw {amount} | Balance left: {self.balance}{self.ANSI_RESET}")
        return True

    def deposit(self, amount: int) -> bool:
        self.balance += amount
        return True